democritus_dates/dates_and_times.py

```diff
@@ -459,3 +459,18 @@ def time_waste(n=3):
     time.sleep(n)
     message = f'I just wasted {n} seconds of your life.'
     print(message)
+
+def time_as_float(time_string: str) -> float:
+    """converts a given HH:MM time string to float"""
+    try:
+        hours, mintues = list(map(int, time_string.split(":"))) #parse given time string
```

Comment:
I suggest fixing a typo in the `minutes` variable name

```suggestion
        hours, minutes = list(map(int, time_string.split(":"))) #parse given time string
```